src/util.py

```diff
@@ -260,6 +260,55 @@ def xr_add_cyclic_longitudes(da, coord):
 
     return new_da
 
+def set_wedge_boundary(ax, lon_range, lat_range, res=1):
+    """
+    Utility function to set the boundary of ax to a wedge shape that is created
+    using given ranges of longitudes and latitudes.
+
+    Args:
+        ax (:class:'matplotlib.axes'):
+            The axes to which the boundary will be applied.
+
+        lon_range (:class:'tuple'):
+            The two-tuple containting the start and end of the desired range of
+            longitudes. The first entry must be smaller than the second entry.
+            Both entries must be between [-180 , 180].
+
+        lat_range (:class:'tuple'):
+            The two-tuple containting the start and end of the desired range of
+            longitudes. The first entry must be smaller than the second entry.
+            Both entries must be between (-90 , 90).
+
+        res (:class:'int'):
+            The size of the incrementation for vertices in degrees. Default is
+            a vertex every one degree of longitude. A higher number results in
+            a lower resolution boundary.
+    """
+    import cartopy.crs as ccrs
+    import matplotlib.path as mpath
+    
+    if (lon_range[0] >= lon_range[1]) :
```

Comment:
How would you define a boundary that passes through -180/180? This would seem to imply that my boundary can't cross the antimeridian.

Reply:
What happens if we want the map to be centered on the Pacific Ocean?